tests/test_matrix_utils.py

```
@@ -6025,6 +6025,27 @@ async def mock_get_displayname(*args, **kwargs):
+class _InlineExecutorLoop:
+    def __init__(self, loop):
+        self._loop = loop
+
+    def is_running(self):
+        return True
+
+    def run_in_executor(self, _executor, func, *args):
+        fut = self._loop.create_future()
+        try:
+            result = func(*args)
+        except Exception as exc:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)